lua/telescope/config.lua

```diff
@@ -831,6 +831,28 @@ append(
     Default: require("telescope.previewers").buffer_previewer_maker]]
 )
 
+append(
+  "git_log_graph",
+  {
+    show = false,
+    all_branches = false,
+    inverted = false,
+  },
+  [[
+  Show git log with --graph option.
+    Args:
+    - show                      showing the graph or not
+        false (default)
+        true
+    - all_branches              showing all branches with graph
+        false (default)
+        true
+    - inverted                  if first git log showing on the top
+        false (default)
+        true
+    Default: {show = false, all_branches = false, inverted = false}]]
+)
```

Comment:
not gonna happen, there is no need to add a global configuration that only affects one builtin.